### DISCUSSION TITLE
How to load level via console command?

### DISCUSSION THREAD

**Discussion by UsualCoder1** [o3de/o3de#2413] [Q&A]
How to load level/map via console command ?
Does the "map" command do not works anymore ?

**Comment by lemonade-dm** [comment]
@sconel  @mbalfour-amzn Would you know how to load a level(I am assuming prefab based)?

I know there is a `LoadLevel` command, but is that all that is needed?

**Reply by mbalfour-amzn** [reply]
Assuming the level is prefab-based, the command looks like "LoadLevel levels/mylevel/mylevel.spawnable".  If the level is still slice-based, the command looks like "LoadLevel mylevel".

**Reply by UsualCoder1** [reply]
in my case level path is - "C:\o3de_master\MyProject\Levels\level1\level1.prefab"
but when i run GameLauncher - "C:\o3de_master\MyProject\build\windows_vs2019\bin\profile\MyProject.GameLauncher.exe"
and type "loadlevel level1" or "loadlevel level1.prefab" or "loadlevel levels\level1.prefab" in debug console - nothing is happened

**Reply by mbalfour-amzn** [reply]
The command that should work in this case is "LoadLevel levels/level1/level1.spawnable".  The path you need to provide is the relative path in the Cache directory to the game-ready version of a prefab, which is a "spawnable".

**Reply by AMZN-alexpete** [reply]
Another thing you can do is add the "LoadLevel levels/level1/level1.spawnable" command to your project's autoexec.cfg so it automatically loads the level when the launcher starts.
